apps/crn-server/src/migrations/16563279171200-move-access-instructions-to-usage-notes.ts

```diff
@@ -0,0 +1,22 @@
+/* istanbul ignore file */
+import { RestResearchOutput } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class MoveResearchOutputAccessInstructionsToUsageNotes extends Migration {
+  up = async (): Promise<void> => {
+    await applyToAllItemsInCollection<RestResearchOutput>(
+      'research-outputs',
+      async (researchOutput, squidexClient) => {
+        await squidexClient.patch(researchOutput.id, {
+          usageNotes: {
+            iv: researchOutput.data.accessInstructions?.iv || '',
+          },
+        });
+      },
+    );
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  down = async (): Promise<void> => {};
```

Comment:
i would add a `down` method implementation to move the data back in case in screws up some pipelines for us or something

Reply:
but the data is not being removed

Reply:
yeah i guess you're right, i thought maybe we should remove it from the new field in the 'down' method but i think having that extra data on the document doesnt actually do anything